src/workspace-hack/Cargo.toml

```diff
@@ -146,13 +147,14 @@ phf_shared = { version = "0.11", features = ["std", "uncased"] }
 postgres-types = { git = "https://github.com/madsim-rs/rust-postgres.git", rev = "87ca1dc", default-features = false, features = ["chrono-04", "derive", "postgres-derive", "with-chrono-0_4"] }
 proc-macro2 = { version = "1", features = ["proc-macro", "span-locations"] }
 prometheus = { version = "0.13", features = ["libc", "process", "procfs", "protobuf"] }
-prost = { version = "0.11", features = ["no-recursion-limit", "prost-derive", "std"] }
+prost-93f6ce9d446188ac = { package = "prost", version = "0.10", features = ["prost-derive", "std"] }
+prost-a6292c17cd707f01 = { package = "prost", version = "0.11", features = ["no-recursion-limit", "prost-derive", "std"] }
```

Comment:
Is this okay? cc @skyzh

Reply:
Then this PR didn't upgrade successfully to the latest version :(